muxers/mplex/src/lib.rs

```diff
@@ -85,6 +85,12 @@ impl MplexConfig {
         self
     }
 
+    /// Sets the frame size used when sending data
+    pub fn split_send_size(&mut self, size: usize) -> &mut Self {
+        self.split_send_size = size;
```

Comment:
This should be limited to the maximum allowed frame size ([currently 1 MiB](https://github.com/libp2p/specs/tree/master/mplex#writing-to-a-stream)).